### PR TITLE
Add double type in GetSpannerColumnType

### PR DESCRIPTION
**Pull request by mayurkale22**
Fixes #<issue_number_goes_here>

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Comment by mayurkale22** [comment]
@TanmayVartak Pls review.

File: utilities/utils.go [hunk 1]
```diff
@@ -168,6 +168,8 @@ func GetSpannerColumnType(cqlType string) string {
 		return "array[string, bool]"
 	case "float":
 		return "float64"
```
Comment (TanmayVartak) [inline utilities/utils.go]: float in Cassandra should map to float32 in Spanner.

**Review by TanmayVartak — CHANGES_REQUESTED** [review]

File: utilities/utils.go (hunk 1 above)
Comment (mayurkale22) [reply utilities/utils.go]: I would like to handle that in the future PRs. This may impact other pieces in the proxy.

Comment (TanmayVartak) [reply utilities/utils.go]: lets add a comment for now otherwise this is confusing.

**Review by TanmayVartak — APPROVED** [review]

File: utilities/utils.go (hunk 1 above)
Comment (mayurkale22) [reply utilities/utils.go]: Done

**Pull request merged**